clusterloader2/pkg/test/simple_test_executor.go

```diff
@@ -57,6 +60,23 @@ func (ste *simpleTestExecutor) ExecuteTest(ctx Context, conf *api.Config) []erro
 			}
 		}
 	}
+
+	for _, summary := range ctx.GetMeasurementManager().GetSummaries() {
+		summaryText, err := summary.PrintSummary()
+		if err != nil {
+			errList = append(errList, fmt.Errorf("printing summary %s error: %v", summary.SummaryName(), err))
+			continue
+		}
+		if ctx.GetClusterLoaderConfig().ReportDir == "" {
+			glog.Infof("%v: %v", summary.SummaryName(), summaryText)
+		} else {
+			filePath := path.Join(ctx.GetClusterLoaderConfig().ReportDir, summary.SummaryName()+"_"+conf.Name+"_"+time.Now().Format(time.RFC3339)+".txt")
```

Comment:
Just so we remember, can you add a TODO above this to make sure summary filenames are kept backward compatible with current naming?

Reply:
Done.